community/tutorials.rst

```diff
@@ -14,31 +14,13 @@ The Godot video tutorials by `GDQuest <https://www.youtube.com/channel/UCxboW7x0
 
 GDQuest's *Learn GDScript From Zero* is a free and open source interactive tutorial for absolute beginners to learn to program with Godot's GDScript language. It is available as a `desktop application <https://gdquest.itch.io/learn-godot-gdscript>`_  or `in the browser <https://gdquest.github.io/learn-gdscript>`_.
 
-Some tutorials mentioned below provide more advanced tutorials, e.g. on 3D or shaders.
+Some tutorials mentioned below cover more advanced subjects, e.g. on 3D or shaders.
 
 Video tutorials
 ---------------
 
-- `Bastiaan Olij <https://www.youtube.com/BastiaanOlij>`_ (3D, AR and VR, GDScript)
-- `BornCG <https://www.youtube.com/playlist?list=PLda3VoSoc_TTp8Ng3C57spnNkOw3Hm_35>`_ (2D and 3D, GDScript)
-- `Clear Code <https://www.youtube.com/watch?v=nAh_Kx5Zh5Q>`_ (2D, GDScript, Programming Basics)
-- `FencerDevLog <https://www.youtube.com/@FencerDevLog>`_ (2D, 3D, GDScript, Shaders)
-- `FinePointCGI <https://www.youtube.com/channel/UCSojAWUnEUTUcdA9iJ6bryQ>`_ (2D, 3D, GDScript and C#)
-- `GDQuest <https://www.youtube.com/channel/UCxboW7x0jZqFdvMdCFKTMsQ/playlists>`_ (2D and 3D, GDScript and C#)
-- `Game Dev Artisan <https://www.youtube.com/@GameDevArtisan>`_ (2D, GDScript)
-- `Game Development Center <https://www.youtube.com/c/GameDevelopmentCenter>`_ (2D, networked multiplayer, GDScript)
-- `Game Endeavor <https://www.youtube.com/channel/UCLweX1UtQjRjj7rs_0XQ2Eg/videos>`_ (2D, GDScript)
-- `Gwizz <https://www.youtube.com/@Gwizz1027>`_ (2D, GDScript)
-- `Godotneers <https://www.youtube.com/@godotneers>`_ (2D, Shaders, GDScript)
-- `HeartBeast <https://www.youtube.com/@uheartbeast>`_ (2D, GDScript)
-- `Malcolm Nixon <https://youtube.com/@MalcolmAnixon>`_ (AR and VR, GDScript)
-- `Muddy Wolf <https://www.youtube.com/@MuddyWolf>`_ (2D, 3D and VR, GDSCript)
-- `KidsCanCode <https://www.youtube.com/channel/UCNaPQ5uLX5iIEHUCLmfAgKg/playlists>`__ (2D and 3D, GDScript)
-- `Maker Tech <https://www.youtube.com/@MakerTech/>`_ (2D, GDScript)
-- `Pigdev <https://www.youtube.com/@pigdev>`_ (2D, GDScript)
-- `Queble <https://www.youtube.com/@queblegamedevelopment4143>`_ (2D, GDScript)
-- `Quiver <https://quiver.dev/>`_ (2D, GDScript)
-- `Snopek Games <https://www.youtube.com/@SnopekGames>`_ (3D, networked multiplayer, AR and VR, GDScript)
+For video tutorials we recommend looking for stuff on YouTube. There's many great
```

Comment:
I guess "Youtube" could be the a hypertext link to https://www.youtube.com/ . Just quality of life.